config.go

```diff
@@ -19,6 +20,50 @@ const (
 	DefaultSkipBloomFilters     = false
 )
 
+const (
+	parquetGoModulePath = "github.com/segmentio/parquet-go"
+)
+
+var (
+	defaultCreatedByInfo string
+	defaultCreatedByOnce sync.Once
+)
+
+func defaultCreatedBy() string {
+	defaultCreatedByOnce.Do(func() {
+		createdBy := parquetGoModulePath
+		build, ok := debug.ReadBuildInfo()
```

Comment:
Is this available on 1.17? Ok by me if it is not

Reply:
It was added in Go 1.12!